server/bleep/src/periodic/remotes.rs

```diff
@@ -138,6 +138,10 @@ async fn update_credentials(app: &Application) {
 }
 
 pub(crate) async fn check_repo_updates(app: Application) {
+    while app.credentials.github().is_none() {
```

Comment:
Could we get stuck in this state?

Reply:
Yes, if there's no authentication we'll never enter this loop.